VaccinationCertificate/ADOPTERS.yaml

```diff
@@ -0,0 +1,10 @@
+description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```

Comment:
could we mention here any project?

Reply:
there isn't any project but created this for the paper.